src/main/java/com/model/Code.java

```diff
@@ -0,0 +1,30 @@
+package com.model;
+
+import java.util.UUID;
+
+public class Code {
+
+    private User user;
```

Comment:
maybe better add Code code in user model?
this class shouldn't depend from user